Makefile

```diff
@@ -21,4 +23,4 @@ db_mongo_prepare: mymongoinit
 test: db_mongo_prepare db_mysql_prepare
 	bash ./scripts/test.sh
 
-.PHONY: mysqlinit mymongoinit createmysqldb dropmysqldn mysqlstop mongostop test
+.PHONY: mysqlinit mymongoinit createmysqldb dropmysqldn mysqlstop mongostop test check
```

Comment:
no need to add all the targets in .PHONY
we use this to specify the targets are considered as phony targets and are not actual file names. It helps to avoid conflicts if a file with the same name as a target exists.

Reply:
ok

Reply:
this is still unaddressed

Reply:
corrected

Reply:
you removed everything, kindly read my first comment in this thread.
here in this case, you should include only `test` and `check` because there are files with the same name as these targets exist.

Reply:
shahzad bhai please tell me are you refering to the scripts files